easypost/services/batch_service.py

```diff
@@ -95,4 +95,14 @@ def get_next_page(
         NOTE: This function has known issues and should not be used.
```

Comment:
please specify what the known issues are - they are known to us but maybe not to users, let's define it here so they can make an educated decision.

Reply:
Resolved